### PR TITLE
XSUP-39590/ XDR IR mappers update

### PR DESCRIPTION
**Pull request by RotemAmit**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-39590)

## Description
Added the severity "Critical" to the `XDR - Incoming Mapper` and `Cortex XDR - Outgoing Mapper`.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by RotemAmit** [comment]
A screenshots of the new mapping.
XDR - Incoming Mapper
xdr incoming mapper update
Cortex XDR - Outgoing Mapper
xdr outgoing mapper update

**Review by MLainer1 — APPROVED** [review]
🧙‍♀️

File: Packs/CortexXDR/ReleaseNotes/6_1_55.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#### Mappers
+##### XDR - Incoming Mapper
+Added support for mapping in the severity **critical**.
```
Comment (Shellyber) [inline Packs/CortexXDR/ReleaseNotes/6_1_55.md:3]:
```suggestion
Added support for **critical** severity mapping from Cortex XDR incident to Cortex XSOAR incident.
```

File: Packs/CortexXDR/ReleaseNotes/6_1_55.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+##### Cortex XDR - Outgoing Mapper
+Added support for mapping out the severity **critical**.
```
Comment (Shellyber) [inline Packs/CortexXDR/ReleaseNotes/6_1_55.md:6]:
```suggestion
Added support for **critical** severity mapping from Cortex XSOAR incident to Cortex XDR incident.
```

**Review by Shellyber — APPROVED** [review]
See RN before merging

**Pull request merged**